x/subscription/types/keys.go

```diff
@@ -21,18 +23,34 @@ const (
 	// prefix for the subscription fixation store
 	SubsFixationPrefix = "subs-fs"
 
-	// prefix for the subscription fixation store
+	// prefix for the subscription timer store
 	SubsTimerPrefix = "subs-ts"
 
 	// prefix for the CU tracker fixation store
 	CuTrackerFixationPrefix = "cu-tracker-fs"
+
+	// prefix for the CU tracker timer store
+	CuTrackerTimerPrefix = "cu-tracker-ts"
 )
 
+// CuTrackerKey encodes a keys using the subscription's consumer address, provider address and the relay's chain ID
 func CuTrackerKey(sub string, provider string, chainID string) string {
 	return sub + " " + provider + " " + chainID
 }
 
+// DecodeCuTrackerKey decodes the CU tracker key
 func DecodeCuTrackerKey(key string) (sub string, provider string, chainID string) {
 	decodedKey := strings.Split(key, " ")
 	return decodedKey[0], decodedKey[1], decodedKey[2]
 }
+
+// DecodeCuTrackerTimerKey decodes the CU tracker key
+func DecodeCuTrackerTimerKey(key string) (sub string, shouldRemove bool) {
```

Comment:
if we use the data in the timer store we dont need this

Reply:
done. see https://github.com/lavanet/lava/commit/93dc7f6261cac796b09559e5fa4c33be54361ac4